static/js/components/inputs/util.js

```diff
@@ -8,7 +8,7 @@ import SpinnerButton from '../SpinnerButton';
  * Helper function to create dialog action buttons, with a SpinnerButton for the save button
  */
 export const dialogActions = (
-  onCancel: Function, onSave: Function, inFlight: bool, text: string='Save', saveClass: string=''
+  onCancel: Function, onSave: Function, inFlight: bool, text: string='Save', saveClass: string='', disabled: bool=false
```

Comment:
is the annotation for `text`, `saveClass` and `disabled` still necessary when they're provided with a default value? I feel like Flow should be able to infer their types from that